src/helpers/xml-builder.js

```diff
@@ -4,14 +4,18 @@
 /* eslint-disable no-case-declarations */
 /* eslint-disable no-plusplus */
 /* eslint-disable no-else-return */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-continue */
+/* eslint-disable eqeqeq */
+/* eslint-disable no-lonely-if */
```

Comment:
I would want these not as a part of same PR.
Reason being I want to see the errors and warning. You are forcefully suppressing the warnings, which I don' like.
We can use `--no-verify` to make a commit, but I still want these errors and warnings to show.